pkg/diff/diff.go

```diff
@@ -583,7 +584,7 @@ func getChunkRows(ctx context.Context, db *sql.DB, schema, table string, tableIn
 	query := fmt.Sprintf("SELECT /*!40001 SQL_NO_CACHE */ %s FROM `%s`.`%s` WHERE %s ORDER BY %s%s",
 		columns, schema, table, where, strings.Join(orderKeys, ","), collation)
 
-	log.Debugf("select data by sql %s, args: %v", query, args)
+	log.Debug("select data", zap.String("sql", query), zap.Any("args", args))
```

Comment:
replace `zap.Any` with `zap.Reflect`?
same in _tidb-binlog/driver/example/mysql/mysql.go_ L91.

Reply:
https://godoc.org/go.uber.org/zap#Reflect
in the document, have a word `Outside tests, Any is always a better choice`, should we replace `Any` to `Reflect`? @kennytm

Reply:
I think this doc can be exchanged to something like
```
Reflect is always a better choice
```
:grimacing:

Reply:
`but it's relatively slow and allocation-heavy`, seems not suggest 
😬

Reply:
@WangXiangUSTC @csuzhangxc 

1. If the type is `interface{}`, `Any` is indeed better than `Reflect` since it could fallback to faster methods if it turns out to be a string/int/etc. However `args` is an `[]interface{}`.
2. When you know none of the predefined methods applies, `Reflect` is a better choice than `Any`, since `Any` will eventually give up and call `Reflect` anyway.
3. Every other method is a better choice than both `Any` and `Reflect`.
4. You could implement [ArrayMarshaler](https://godoc.org/go.uber.org/zap/zapcore#ArrayMarshaler) to use `zap.Array`, for instance.

Reply:
I do a bench mark test for []interface{}, reflect is a little better than any
BenchmarkAnyLog-4                 500000             15240 ns/op
BenchmarkReflectLog-4             500000             15520 ns/op

Already changed, PTAL again